packages/browser/src/integrations/graphqlClient.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  SEMANTIC_ATTRIBUTE_HTTP_REQUEST_METHOD,
+  SEMANTIC_ATTRIBUTE_SENTRY_OP,
+  SEMANTIC_ATTRIBUTE_URL_FULL,
+  defineIntegration,
+  spanToJSON,
+} from '@sentry/core';
+import type { Client, IntegrationFn } from '@sentry/types';
+import { parseGraphQLQuery } from '@sentry/utils';
+
+interface GraphQLClientOptions {
+  endpoints: Array<string>;
```

Comment:
m: Instead of `string[]`, let's type this as `(string | RegExp)[]`. We also use this pattern for e.g. `tracePropagationTargets`. Then, instead of doing `endpoints.includes(url)`, you can use the `stringMatchesSomePattern(url, endpoints)` utility to check these :)